src/controllers/payments.js

```diff
@@ -140,9 +141,11 @@ exports.put = function *(id) {
 
   payment.id = id.toLowerCase()
 
-  let isPrepared = yield Payments.validate(payment)
-  if (isPrepared) {
-    yield Payments.settle(payment)
+  yield Payments.validate(payment)
+  const truePayment = settlementQueue.storePayment(payment)
```

Comment:
The name `truePayment` doesn't really give me any idea how it differs from `payment`, is there another name that describes the difference between the two?